portal/src/main/webapp/js/src/study-view/component/MutatedGeneCNATable.js

```diff
@@ -218,6 +218,14 @@ var EnhancedFixedDataTableSpecial = (function() {
                     var list = ($("#hide_column_checklist").dropdownCheckbox("items"));
                     hideColumns(list);
                 });
+            // add title attributes to unlabeled inputs generated by
+            // .dropdownCheckbox() for accessibility, until
+            // https://github.com/Nelrohd/bootstrap-dropdown-checkbox/issues/33 is
+            // fixed upstream
+            $('#hide_column_checklist input[type="checkbox"].checkbox-all')
+                .attr('title', 'Select all');
+            $('#hide_column_checklist input[type="text"].search')
+                .attr('title', 'Search');
```

Comment:
Yes, you did! I think these changes also will benefit the general EFDT library and that's why I asked you sent to the master branch. We have two cbioportal specific designed branches study-view-summary and study-view-clinical. Everytime if any changes made to master, I will rebase these two branches and update the library in our cbioportal repository.